social/src/Components/BottomNav/BottomNav.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './BottomNav.css';
+import { IconButton } from '@mui/material';
+import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
+import ExploreRoundedIcon from '@mui/icons-material/ExploreRounded';
+import NotificationsRoundedIcon from '@mui/icons-material/NotificationsRounded';
+import BookmarkRoundedIcon from '@mui/icons-material/BookmarkRounded';
+import PersonRoundedIcon from '@mui/icons-material/PersonRounded';
+import SettingsApplicationsRoundedIcon from '@mui/icons-material/SettingsApplicationsRounded';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+const BottomNav = () => {
+    let navigate = useNavigate();
+    const { theme } = useSelector(state => state.theme);
+    const getActiveStyles = ({isActive}) => 
+        isActive ? {
+            color: "white",
+            borderRadius: "50%",
+            backgroundColor: "var(--primary)"
+        } : {}
```

Comment:
Can we extract this function out as a utility. Would be easy to understand and apply changes if required!!

Reply:
Okay will do the changes